src/main/java/mate/controller/InjectDataController.java

```diff
@@ -0,0 +1,52 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.DriverService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/initializeDB")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/inject")
```

Reply:
it's a bad practice to use camelCase here. Also, why do use here annotation and in other cases web.xml? It's not consistent, choose one option